{
    "original_source": {
        "database(url)": "https://github.com/secureIT-project/CVEfixes",
        "software_system(CPE)": [
            "cpe:2.3:a:sqlite:sqlite:3.30.1:*:*:*:*:*:*:*",
            "cpe:2.3:o:canonical:ubuntu_linux:16.04:*:*:*:lts:*:*:*",
            "cpe:2.3:o:canonical:ubuntu_linux:18.04:*:*:*:lts:*:*:*",
            "cpe:2.3:o:canonical:ubuntu_linux:19.10:*:*:*:*:*:*:*"
        ],
        "fix_commit": "https://github.com/sqlite/sqlite/commit/d8f2d46cbc9925e034a68aaaf60aad788d9373c1",
        "version_end_excluding": {
            "sqlite:sqlite": "unknown",
            "canonical:ubuntu_linux": "unknown"
        },
        "path": {
            "file_path": "manifest.uuid"
        }
    },
    "programming_language": "sql",
    "licences": {
        "database": "MIT",
        "affected_repository": "Other"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2019-19959"
        ],
        "CWE": [
            "NVD-CWE-Other"
        ]
    }
}